src/main.cpp

```diff
@@ -45,5 +50,30 @@ void mainLoop(void)
     // TODO: link module exited, attempt recovery
   }
 
-  // TODO: Failure actions (alarm, watchdog)
+  // Scan through all alarms, looking for any set warning that need to be addressed
+  struct alarmProperties properties = {0};
+  if (alarmCheckAll(&properties)) {
```

Comment:
Does the alarm need to be initialized before checked?

Reply:
Any alarm that has been initialized via `alarmInit` will be checked by this function and the aggregate alarm properties that we need to deal with will be loaded into `properties`; so this isnt checking a specific alarm